lost/src/main/java/com/mapzen/android/lost/api/Status.java

```diff
@@ -67,11 +71,10 @@ public Status(int statusCode, PendingIntent pendingIntent) {
    * @param requestCode associated with activity.
    * @throws IntentSender.SendIntentException
    */
-  public void startResolutionForResult(Activity activity, int requestCode)
+  public void startResolutionForResult(final Activity activity, final int requestCode)
```

Comment:
Perhaps the best course of action is to update the Javadoc for now and create an issue to implement the wrapper activity that can make an additional settings request and set `RESULT_OK` if conditions have been met at a later time.